packages/synapse-react-client/src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -199,246 +214,224 @@ export const CreateOAuthModal: React.FunctionComponent<
+      ))}
+    </StyledFormControl>
+  )
 
-                          {redirectUris.length - 1 === idx && (
-                            <Button
-                              onClick={handleRedirectUriAdd}
-                              disabled={singleUri.uri.length === 0}
-                            >
-                              Add Uri
-                            </Button>
-                          )}
-                        </div>
-                      ))}
-                    </Col>
-                    <Col lg={6} md={6} sm={12} xs={12}>
-                      <Form.Label>Sector Identifier URI</Form.Label>
-                      <Form.Control
-                        onChange={e => setSectorUri(e.target.value)}
-                        placeholder="https://"
-                        type="text"
-                        value={sectorUri}
-                      />
-                    </Col>
-                  </Row>
+  const content = (
+    <>
+      {isLoadingUpdate && (
+        <div className={'SRC-center-text'}>
+          <SynapseSpinner size={40} />
+        </div>
+      )}
+      {!isLoadingUpdate && (
+        <>
+          <Typography variant="body1" mb={'10px'}>
+            To protect you and your users, your consent screen and application
+            will need to be verified by Sage Bionetworks. Before your consent
+            screen and application are verified by Sage Bionetworks, you can
+            still test your application with limitations.
+          </Typography>
 
-                  <button
-                    className="delete-button"
-                    onClick={() => {
-                      setIsDelete(true)
-                      setIsShowingConfirmModal(true)
-                    }}
-                  >
-                    <IconSvg icon="delete" sx={{ color: 'error.main' }} />
-                    DELETE CLIENT
-                  </button>
-                </div>
-              )}
-            </>
+          {isEdit && (
+            <Typography sx={{ mt: '16px' }} variant="label">
+              Client ID: {client?.client_id}
+            </Typography>
           )}
-        </Modal.Body>
-        {error && <Alert variant="danger">{error?.reason}</Alert>}
-        <Modal.Footer>
-          <Button
-            variant="default"
-            onClick={() => {
-              hide()
-              setError(undefined)
-            }}
+          <Grid
+            container
+            rowSpacing={GRID_ROW_SPACING}
+            columnSpacing={GRID_COL_SPACING}
           >
-            CANCEL
-          </Button>
-          <Button variant="primary" onClick={onCreateClient}>
-            Save
-          </Button>
-        </Modal.Footer>
-      </Modal>
+            <Grid item md={GRID_WIDE} xs={GRID_NARROW}>
+              <TextField
+                label="Client Name"
+                required
+                onChange={e => setClientName(e.target.value)}
+                placeholder="Client Name"
```

Comment:
SRC TextField doesn't display placeholder text until the input field is focused, whereas MUI TextField displays placeholder text in all states until new text is entered into the field. Is one of these styles preferred? If so, I can work to align the placeholder text handling between the TextField components

Reply:
Sounds like a bug in SRC TextField. Placeholder text should always be shown if there's no value